models/user.js

```diff
@@ -1,27 +1,33 @@
-'use strict';
-const {
-  Model
-} = require('sequelize');
-module.exports = (sequelize, DataTypes) => {
-  class User extends Model {
-    /**
-     * Helper method for defining associations.
-     * This method is not a part of Sequelize lifecycle.
-     * The `models/index` file will call this method automatically.
-     */
-    static associate(models) {
-      // define association here
-    }
-  };
-  User.init({
-    nickname: DataTypes.STRING,
-    avatarUrl: DataTypes.STRING,
-    email: DataTypes.STRING,
-    pointsForNotGiving: DataTypes.INTEGER,
-    meetingsCount: DataTypes.INTEGER
-  }, {
-    sequelize,
-    modelName: 'User',
-  });
-  return User;
-};
\ No newline at end of file
+const Sequelize = require('sequelize');
```

Comment:
This has been done on other branch and is awaiting acceptance on another PR, You can check out this branch and remove this file content